.travis.yml

```diff
@@ -13,7 +13,8 @@ addons:
             - libexpat1-dev
 
 install: make compile
-script: make test
+script:
+    - make test dialyzer
```

Comment:
I think it be better to run test and dialyzed separately. The log in Travis will be a bit easier to analyse IMO.